### DISCUSSION TITLE
Whether the "arrow" files storage the result of ArchR projecet after perfoming filterDoublets function or not？

### DISCUSSION THREAD

**Discussion by Jane8166** [GreenleafLab/ArchR#1889] [Questions / Documentation] [answered]
Hello everybody,

I filtered the cells with lower TSS and nFrags , and computed the  Doublets score , and storaged the results as "project1";then I filtered the Doublets and saved the result as "project2".The cell number of project2 is lower the project1. But when I use the Arrowfiles from "project2" to create the new ArchR project "project3", the cell number of project3 is equal to the cell number of project1, what happened? 

Thanks for your reply.

**Comment by rcorces** [comment, marked as answer]
If you look at the code for `filterDoublets()`, this works at the level of the `ArchRProject` and does not touch the Arrow files themselves.
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/FilterCells.R#L45-L106

**Reply by Jane8166** [reply]
Thank you very much, I used the arrow of "project1" and refiltered the doublets, and this question has been resolved.
